src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -255,13 +283,23 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
           renderTableContents(item).map((content, index) => {
             const cellProps: TableCellProps = isReactNode(content) ? { children: content } : content;
             const headCell = renderTableHeader?.[index];
+            const cellSize = this.cellSizes[index];
 
             if (copyClassNameFromHeadCells && headCell) {
               cellProps.className = cssNames(cellProps.className, headCell.className);
             }
 
+            if (cellSize !== undefined) {
+              cellProps.size = cellSize;
+            }
+
+            if (resizable) {
+              cellProps._onResize = width => this.handleCellResize(index, width);
+              cellProps._onResizeComplete = () => this.persistCellSizes();
+            }
```

Comment:
Given that there is an `isResizable` prop these should be able to be passed in unconditionally.